### DISCUSSION TITLE
Permissions for pushing to branches of main repo

### DISCUSSION THREAD

**Discussion by sariths** [LBNL-ETA/pyradiance#25] [General]
@taoning I tried creating a branch on the main repo (https://github.com/LBNL-ETA/pyradiance/tree/basicExamples) and pushing to it. But it seems that I don't have the required permissions to push. So, I am working with branch from my own fork. 
![image](https://github.com/LBNL-ETA/pyradiance/assets/10361848/ef7e3198-beb0-4cf1-97d9-a9964cf87551)
I am fine (and would prefer) syncing my local fork to main then branching off of that (https://github.com/sariths/pyradiance/tree/testFilesBasicExamples) . This note is just for future reference. 
(This is not really an issue/bug/feature etc, so posting here in discussions).

**Comment by taoning** [comment]
Did you check for your local GitHub authentication and ssh key issue? You should have permission to write.

**Reply by sariths** [reply]
I can push to the branch from my own fork, so I am not sure where the issue lies. Anyway, I would prefer doing this way and just sending PRs to the main branch once I am done.
